build/image_checks/checkImagesNamingConvention.js

```diff
@@ -0,0 +1,110 @@
+const path                    = require('path');
+const fs                      = require('fs');
+const common                  = require('../../scripts/common');
+const {
+    getDefaultPath,
+    getImageReferences,
+    gitIgnoreToArray,
+    flatten,
+    fetchFiles,
+    renameFiles,
+    validateFilename,
+    convertToSnakeCase }      = require('../utils/cli');
+
+const ignored_files = gitIgnoreToArray();
```

Comment:
@easteregg 
It seems that the `ignore-files` doesn't affect the result.
![image](https://user-images.githubusercontent.com/16184705/50505697-243cd880-0ab0-11e9-8e74-d45ca30b5250.png)

Reply:
I think I didn't have any .DS_Store locally. I'll push a fix.